### ISSUE TITLE
selection of field values in popup windows

### ISSUE THREAD

**Issue by anvalanz** [tomchadwin/qgis2web#707]

**Comment by anvalanz** [comment]
It seems not possibile to select any value from the popup window.
Is it a bug or normal ?

Antonio Valanzano

**Comment by boesiii** [comment]
By default all of your values in your attribute table should be available in the popup window.  Are you not seeing something similar to the image below?  The items circled in blue are all the attributes.
![fields2](https://user-images.githubusercontent.com/2079638/52283270-28220d00-2930-11e9-9a0c-45704d873cf0.PNG)

**Comment by anvalanz** [comment]
Thanks for the answer.
Sorry but i didn't correctly explain the problem.
I was talking about the possibility of selecting 
 field values in the popup window  which shows up when I click on a feature in a map.

![popup_window](https://user-images.githubusercontent.com/6723429/52334054-24a98700-29ff-11e9-8791-8ff8433f5b46.png)

Regards 
Antonio Valanzano

**Comment by tomchadwin** [comment]
I can highlight OK here:

![2019-02-06 10_12_01-mozilla firefox](https://user-images.githubusercontent.com/7840213/52334515-ccbb5200-29f7-11e9-80d1-fe4212f1c2cf.png)

What OS/browser are you using - perhaps that might have an effect?

**Comment by anvalanz** [comment]
I have found the same problem on my 2 pcs with:
 Win10 home ( 64 bit) / Firefox 65.0 (64 bit)
Win7 home premium (64 bit) / Firefox 65.0 (64 bit)
Win7 home premium (64 bit) / Chrome 72.0 (64 bit)

the pages were created using the Openlayers option of the plugin

Thanks in advance for the assistance.
AV

**Comment by boesiii** [comment]
I am using Windows 7 with Chrome 72 and I am not having the problem.  Why are you trying to highlight?  What do you plan to do after you highlight?  Copy and paste?

**Comment by boesiii** [comment]
Are you able to highlight other web pages like google.com or espn.com?

**Comment by anvalanz** [comment]
Thanks for the reply.

My intention is to allow users to copy and past part of the infos which are
displayed in the popup window.

I have made several others attempts but the situation is always the same.

Please find attached two screenshots: one from the preview (in QGIS) with
the possibility to highlight infos and the other from the exported files
with no possibility to highlight infos.
I have uploaded the result to the following link:
http://sleepy-knuth-a28d5a.netlify.com/ and you can try and see the result
(the link is active only for 24 hours).

Then I have tried to export with the leaflet option and with this option it
is possible to highlight infos, see the attached screenshot.

In conclusion it seems a problem with openlayers export.
Let me know something.

AV

ps. I am using QGIS 2.18.28 LTR on Windows 7 installed from OSGeo4W

Il giorno mer 6 feb 2019 alle ore 18:59 boesiii <notifications@github.com>
ha scritto:

**Comment by boesiii** [comment]
I don't know why yours isn't working correctly.  Make sure you are using the latest version of gis2web.  When I am having issues with q2w I will usually start with one layer and test.  Add more data, test again.

**Comment by anvalanz** [comment]
I have made several new attempts following your suggestion but the situation has not changed.
My configuration is Win7 with  QGIS 2.18.28 and Qgis2Web 2.34.0
The final conclusion is: 
export with Openlayes No highlight
export with LeafLet  YES highlight

Because of this problem I have installed 
QGIS 3.4.4 and Qgis2WEb 3.4.0
and the highlight is possible with both Openlayers AND LeafLet.

It is now up to you to inform users that version 2.34.0 has this problem and therefore it is advisable  to switch to version 3 of QGIS and related version of your plugin.

Thanks a lot for the time you have spent on this problem.
Antonio Valanzano

**Comment by tomchadwin** [comment]
Perhaps this was related to the OpenLayers version - qgis2web3 uses OL5, but I think qgis2web 2 uses OL4. This shouldn't be an issue for long, as QGIS 2 is about to go out of support. Everyone will start to upgrade to QGIS3, with luck.
